example/curlz/README.md

```diff
@@ -0,0 +1,115 @@
+```
+
+# Setup a cURLz to a non-zitified endpoint
+These steps will configure the service using the OpenZiti CLI. At the end of these steps you will have created:
+* a service called `www.google.com`
+* an identity to connect to (dial) the service
+* the service config to connect the service to the overlay
+* the service policies required to authorize the identities for bind and dial
+
+Steps:
+1. log into Ziti. The host:port and username/password will vary depending on your network.
+
+       ziti edge login localhost:1280 -u admin -p admin
+1. Run this script to create everything you need.
+
+       echo Create the service config
+       ziti edge create config www.google.com.hostv1 host.v1 '{"protocol":"tcp", "address":"www.google.com","port":'443'}'
+
+       echo Create the service
+       ziti edge create service www.google.com --configs "www.google.com.hostv1"
+       
+       echo Create an identity to make the dial request and enroll it
+       ziti edge create identity user curlz -a clients -o curlz.jwt
+       ziti edge enroll --jwt curlz.jwt
+       
+       echo Create service policies
+       ziti edge create service-policy www.google.com.dial Dial --service-roles '@www.google.com' --identity-roles '#clients'
+       ziti edge create service-policy www.google.com.bind Bind --service-roles '@www.google.com' --identity-roles '@MacBook-Pro.local-edge-router'
+       
+       echo Create edge router policies
+       ziti edge create edge-router-policy curlz-edge-router-policy --edge-router-roles '#all' --identity-roles '#clients,#servers'
+       ziti edge create service-edge-router-policy curlz-service-edge-router-policy --edge-router-roles '#all' --service-roles '@www.google.com'
+       
+       echo Run policy advisor to check
+       ziti edge policy-advisor services
+1. Run the cURLz example for www.google.com
+
+       ./curlz https://www.google.com curlz.json
+
+## Example Output
+The following is the output you'll see from the cURLz request to www.google.com.
+```
+$ ./curlz https://www.google.com curlz.json
+<!doctype html><html itemscope="" itemtype="http://schema.org/WebPage" lang="en">
+... <a lot of html code>
+</body></html>
+```
+
+# Setup a cURLz to a zitified endpoint
+These steps will utilize the service and identities created in simple-server to provide an example of using cURLz with a zitified endpoint
+
+Steps:
+1. Follow all steps in the simple-service example up to, and including, running the server but **do not** enroll the 
+`simple-client` identity with the Ziti Desktop Edge client. We will do that with the CLI for this example
+1. Open a new terminal and cd into the curlz directory
+
+       cd <repo-root-dir>/sdk-golang/example/curlz/
+1. Run this script to create everything you need.
+
+       echo Copy the identity jwt into the current working directory
+       cp ../simple-server/simple-client.jwt .
+
+       echo Enroll the simple-client identity
+       ziti edge enroll --jwt simple-client.jwt
+
```

Comment:
The reference to `www.google.com` on this line and line 87 seem like they should be replaced with something about simple server

Reply:
it's probably better to replace all the 'www.google.com` with anything else, i'd agree... and then call out specifically that the service is offloading from the overlay at the router.